docs/index.md

```diff
@@ -114,7 +114,29 @@ If the issuer has more than one certificate for the same public key (e.g. partic
 
 Issuers SHALL publish their public keys as JSON Web Key Sets (see [RFC7517](https://tools.ietf.org/html/rfc7517#section-5)), available at `<<iss value from JWS>>` + `/.well-known/jwks.json`, with [Cross-Origin Resource Sharing (CORS)](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin) enabled.
 
-The URL at `<<iss value from JWS>>` SHALL use the `https` scheme and SHALL NOT include a trailing `/`. For example, `https://smarthealth.cards/examples/issuer` is a valid `iss` value (`https://smarthealth.cards/examples/issuer/` is **not**).
+The URL at `<<iss value from JWS>>` SHALL use the `https` scheme, and SHALL use the following TLS version 1.2 or 1.3 configurations:
```

Comment:
I'm uncertain of the intent of this sentence. If a webserver support TLS 1.0, 1.1, 1.2, and 1.3, is that webserver in compliance with this requirement? Could you please clarify?

Generally, are you trying to define a floor or a ceiling? They proposed phrase: "support at least one" instead of "use", below, implies a floor, such that the connection could be insecure if the client accesses the public key doesn't also support these best practices / requirements. 

This PR really begs the question -- why are we putting normative requirements on the server instead of the client?

Reply:
Re: guidance for servers vs clients: if a server doesn't meet these requirements, *nothing* a client does can achieve an appropriate level of protection. So these guidelines are intended to ensure that clients *can* obtain keys in a secure fashion. Clients don't need to retrieve keys directly from an issuer (e.g., they could rely on an intermediary), so I'm not sure it makes sense to impose additional constraints on that side of the interaction.

Re: "ceiling" vs "floor", these terms can be a bit confusing; I *think* the intent of this langauge is to ensure at least one of TLS 1.2 or 1.3 is supported. (Possibly we also want to prohibit servers from supporting TLS 1.0, but that's another question.)

As such, I'd suggest re-wording as:

```suggestion
The URL at `<<iss value from JWS>>` SHALL use the `https` scheme, and SHALL support at least one of the following TLS configurations:
```

Reply:
Superseded by ref to BCP 195 (in line with the "at least" suggestion)